astacus/node/snapshotter.py

```diff
@@ -101,27 +101,30 @@ def _snapshotfile_from_path(self, relative_path):
         st = src_path.stat()
         return SnapshotFile(relative_path=relative_path, mtime_ns=st.st_mtime_ns, file_size=st.st_size)
 
+    def _files_are_the_same(self, old: SnapshotFile, new: SnapshotFile) -> bool:
```

Comment:
nit: This should be probably something in SnapshotFile itself.

Reply:
:heavy_check_mark: done